server/server.js

```diff
@@ -223,7 +223,11 @@ function handleRequest(request, response) {
 
     case "": // Index page
       logRequest(request);
+      if (!config.DEFAULT_BOARD)
         indexTemplate.serve(request, response);
+      else
```

Comment:
the double negation is hard to process. Maybe invert the if and else: `if (config.DEFAULT_BOARD) ... else ...`